libs/host/Configuration/Options.cs

```diff
@@ -339,6 +339,10 @@ internal sealed class Options
         [Option("maxthreads", Required = false, HelpText = "Maximum worker and completion threads in thread pool, 0 uses the system default.")]
         public int ThreadPoolMaxThreads { get; set; }
 
+        [IntRangeValidation(0, int.MaxValue)]
+        [Option("connection-limit", Required = false, Default = 0, HelpText = "Maximum number of client connections")]
```

Comment:
Reword: `Maximum number of simultaneously active client connections.` (note also: added period at the end)